app/models/events/answer_set.rb

```diff
@@ -1,24 +1,26 @@
 module Events
   class AnswerSet < Event
-    event_attributes question_id: :string,
+    event_attributes domain_id: :string,
```

Comment:
excuus, ik bedoel: voeg het attribuut toe aan deze regel:
```
  %w(text intro answer_options).each do |r_attribute|
```

P.S. Realiseer me nu trouwens dat de define_method in dit geval overengineering is (alles kan met delegate to data_from_r gedaan worden), maar dat is iets voor een ander ticket